dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetVersionDifference.java

```diff
@@ -753,80 +805,78 @@ public void setMultiple(boolean multiple) {
 
     public class DatasetReplaceFileItem {
 
-        public DatasetFileDifferenceItem getFdi() {
-            return fdi;
-        }
-
-        public void setFdi(DatasetFileDifferenceItem fdi) {
-            this.fdi = fdi;
-        }
+        private FileSummary fileSummary1;
```

Comment:
I think the convention of naming it old/new would be more descriptive.

Reply:
:+1: